tests/greenheart/test_hydrogen/test_greenheart_system.py

```diff
@@ -321,27 +337,44 @@ def test_simulation_wind_battery_pv_onshore_steel_ammonia(subtests):
 
     # TODO base this test value on something
     with subtests.test("lcoh"):
-        assert lcoh == approx(3.0191128494981223)
+        assert lcoh == approx(3.0191128494981223, rel=rtol)
 
     # TODO base this test value on something
     with subtests.test("lcoe"):
-        assert lcoe == approx(0.034676133587257206)
+        assert lcoe == approx(0.034676133587257206, rel=rtol)
 
     # TODO base this test value on something
     with subtests.test("steel_finance"):
         lcos_expected = 1340.641495821553
 
-        assert steel_finance.sol.get("price") == approx(lcos_expected)
+        assert steel_finance.sol.get("price") == approx(lcos_expected, rel=rtol)
 
     # TODO base this test value on something
     with subtests.test("ammonia_finance"):
         lcoa_expected = 1.0405052843769131
 
-        assert ammonia_finance.sol.get("price") == approx(lcoa_expected)
+        assert ammonia_finance.sol.get("price") == approx(lcoa_expected, rel=rtol)
 
 def test_utilities(subtests):
```

Comment:
Let's make a new test file `test_greenheart_utilities` and then in the next PR we can include the timeseries length checks we discussed there.

Reply:
I've done both of these, but they will be in the next PR

Reply:
I agree. I will do this for the next PR though (which is basically ready now)